camera.py

```diff
@@ -22,9 +24,12 @@ def open(self, filename):
             raise ValueError(f'Failed opening video capture device "{filename}"!')
 
     def convert_frame_color(self, frame):
-        if self.color_conversion_code is None:
-            raise AttributeError('Color conversion code must be set before '
-                                 'converting frame colors!')
+        
+        if self.useISP == True:
+            self.color_conversion_code = cv.COLOR_YUV2RGB_YUYV
+        else:
+            self.color_conversion_code = cv.COLOR_BAYER_GB2RGB
+        
```

Comment:
Don't modify the color conversion code here, set the correct color conversion code in `CameraVM016`'s `__init__` function.